frontend/projects/upgrade/src/app/core/experiments/store/experiments.reducer.ts

```diff
@@ -90,9 +91,18 @@ const reducer = createReducer(
   on(experimentsAction.actionGetExperimentByIdSuccess, (state, { experiment }) =>
     adapter.upsertOne(experiment, { ...state, isLoadingExperiment: false })
   ),
-  on(experimentsAction.actionDeleteExperimentSuccess, (state, { experimentId }) =>
-    adapter.removeOne(experimentId, state)
-  ),
+  // Experiment Delete Actions
```

Comment:
there isn't any existing real delete action? odd.

Reply:
There is one. This PR makes changes to it.